modules/default/weather/providers/openweathermap.js

```diff
@@ -17,10 +17,13 @@ WeatherProvider.register("openweathermap", {
 	defaults: {
 		apiVersion: "3.0",
 		apiBase: "https://api.openweathermap.org/data/",
-		weatherEndpoint: "", // can be "onecall", "forecast" or "weather" (for current)
+		// weatherEndpoint is "onecall" since API 3.0
+		// "onecall", "forecast" or "weather" only for pro customers
+		weatherEndpoint: "onecall",
```

Comment:
Shouldn't this be "/onecall" ?

```suggestion
		weatherEndpoint: "/onecall",
```

Reply:
yes, see my comment above - but we have to change more things to get the tests working